_posts/2019-06-20-2019-ugic-annual-conference-recap.md

```diff
@@ -0,0 +1,101 @@
+
+*Exceptional Attendance*: This year’s conference was exceptionally well attended, with a total of 263 attendees (including both registrants and vendors---possibly the highest number of vendors the conference has seen), 121 students in two days of preconference training (One section of the ArcGIS Pro preconference training even sold out, which is the first time, that UGIC knows of, that a class has been completely filled!), and over 200 people at the Thursday evening social on the Heber Valley Railroad. 
+
+[*Mobile Agenda*](https://website.webmobi.com/ugic2019): UGIC first used a conference app at the 2018 conference held in Vernal. The 2019 conference was the first UGIC conference where there wasn’t a paper agenda booklet in addition to any digital materials, and this was incredibly successful. The conference app provided attendees with easy-to-use venue maps and allowed users to make agenda changes on the fly as needed. 
+
+*Other*: A live band performed during the Wednesday evening vendor social. There were also a lot of friends and family who joined Thursday evening’s BBQ dinner and train ride.
+
+### What Was the Same?
+{: .text-left}
+
+Some aspects of the conference are consistent from year to year, for instance: the fantastic presentations from the Utah GIS community. Also, the UGIC conference is a great time for all attendees! Between the excellent sessions, the vendor social, the indoor golf tournament, the BBQ dinner, and train ride, there was something for everyone. 
+
+Another consistent part of the conference is a great venue in one of Utah’s great cities. This year’s Zermatt Resort was a fantastic venue. There was ample room for all of the conference activities, including the keynote presentations, breakouts, preconference training classes, vendor social, map and poster gallery, and indoor golf tournament. The Zermatt was also close by for the off-site BBQ dinner and train ride at the Heber Valley Railroad.
+
+## To Sum It Up
+{: .text-left}
+
+Utah has been recognized nationally as a great place for GIS innovations, data sharing, and collaboration.  The presentations at the UGIC conference each year highlight the fact that Utah is a great place for anyone using GIS. In fact, some members are now involved in the GIS field in other states, but they value the connections at UGIC so much that they keep coming back for the conference. This is a testament to the quality of GIS in Utah and why the UGIC conference is valuable locally, regionally, and nationally.
+
+[^1]:  Malotki, Ekkehart. *Kokopelli: The Making of an Icon*. (Lincoln, Nebraska: University of Nebraska Press). p. 1. Accessed via [Google Books](https://books.google.com/books?hl=en&lr=&id=qqOgjiqa7R4C&oi=fnd&pg=PR8&dq=kokopelli&ots=MXUc6Y_0p3&sig=wnVth6vhV-52Jpy4CliANmeSBpA#v=onepage&q=the%20overriding%20reason&f=false) on June 20, 2019.   
+[^2]:  Malotki. *Kokopelli: The Making of an Icon*. p 4. 
+(http://americanhistory.si.edu/collections/search/object/nmah_761634).
```

Comment:
this needs to go on the same line as the one before or it doesn't render correct